<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimpleGPTQtDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="chatGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>611</height>
     </rect>
    </property>
    <property name="title">
     <string>对话区</string>
    </property>
    <widget class="QLineEdit" name="chatLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>580</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="chatTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>381</width>
       <height>551</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="chatPushButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>580</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="createImgGroupBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>541</width>
      <height>581</height>
     </rect>
    </property>
    <property name="title">
     <string>图片生成区</string>
    </property>
    <widget class="QLineEdit" name="createImgLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>431</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="createImgPushButton">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>生成</string>
     </property>
    </widget>
    <widget class="QLabel" name="createImgLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>512</width>
       <height>512</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="apiKeyLineEdit">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>600</y>
      <width>471</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="apiKeyLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>600</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>API Key：</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
